### PR TITLE
initial migrations

### PR DESCRIPTION
**Pull request by KMDOFNI**
Industrial Court initial migrations

### CODE REVIEW

**Review by johangant — COMMENTED** [review]
Various points of consideration but overall looks okay to me :+1:

File: project/sites/industrialcourt/modules/custom/industrialcourt_migrations/config/install/migrate_plus.migration.upgrade_d7_file.yml [hunk 1]
```diff
@@ -25,13 +25,18 @@ process:
     -
       plugin: get
       source: filename
+  ammended_filepath:
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/sites/industrialcourt/modules/custom/industrialcourt_migrations/config/install/migrate_plus.migration.upgrade_d7_file.yml:28]: This does appear to be a typo

File: project/config/industrialcourt/config/field.field.node.decision.field_attachments.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description: 'Attach any applications decisions and declarations made in the case using this field'
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:file'
+  handler_settings: {  }
+  file_directory: decisions
+  file_extensions: 'txt doc docx xls xlsx pdf ppt pptx pps ppsx odt ods odp'
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/config/industrialcourt/config/field.field.node.decision.field_attachments.yml:24]: A fair point, probably not to be honest.

File: project/config/industrialcourt/config/user.role.authenticated.yml [hunk 3]
```diff
@@ -3,27 +3,46 @@ langcode: en
...
     - node
+    - path
+    - pathauto
     - search
     - system
 _core:
   default_config_hash: zLyQwFO06-pj9Gj5UZMWuk_aAuAOSC8YqFEHItUk-Tc
 id: authenticated
-label: 'Authenticated user'
+label: 'authenticated user'
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/config/industrialcourt/config/user.role.authenticated.yml]: This is now fixed

File: project/config/industrialcourt/config/system.menu.menu-site-navigation.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+uuid: 974fd10e-43d5-4c2e-9164-548098f65778
+langcode: en
+status: true
+dependencies: {  }
+id: menu-site-navigation
+label: 'Site Navigation'
+description: 'Used on the right side of pages for site navigation'
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/config/industrialcourt/config/system.menu.menu-site-navigation.yml]: This is now fixed

File: project/config/industrialcourt/config/field.storage.node.body.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 uuid: f965a15e-0663-4504-9c5e-a336f3c76d9d
-langcode: en
+langcode: und
```
[the remark this answers is not in the document]
Comment (KMDOFNI) [reply project/config/industrialcourt/config/field.storage.node.body.yml]: This is now fixed

**Review by johangant — APPROVED** [review]
LGTM. Thanks @KMDOFNI

**Pull request merged**